frontend/apps/core-app/src/screens/RecipientListScreen/RecipientListScreen.styles.ts

```diff
@@ -0,0 +1,4 @@
+import styled from 'styled-components/native';
+import { Box } from 'native-base';
+
+export const Container = styled(Box)``;
```

Comment:
I know we said to do this in the style guide. Just wondering if it's really necessary.

Reply:
Whether the entire file is needed

Reply:
I'm assuming that we'll add more styling to the container later, so I'd keep it for now